aptos-move/move-examples/fungible_asset/multisig_managed_coin/sources/multisig_managed_coin.move

```diff
@@ -0,0 +1,231 @@
+    const ENOT_OWNER: u64 = 1;
+
+    /// The asset symbol which should be customized.
+    const ASSET_SYMBOL: vector<u8> = b"MEME";
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    /// Hold refs to control the minting, transfer and burning of fungible assets.
+    struct ManagingRefs has key {
+        mint_ref: MintRef,
+        transfer_ref: TransferRef,
+        burn_ref: BurnRef,
+    }
+
+    fun init_module(creator: &signer) {
+        let multisig_address = multisig_account::get_next_multisig_account_address(signer::address_of(creator));
+        // You should customize those arguments per your need.
+        multisig_account::create_with_owners(
+            creator,
+            vector[@0x123, @0x456],
+            2,
+            vector[],
+            vector[],
+        );
+
+        let constructor_ref = &object::create_named_object(creator, ASSET_SYMBOL);
+
+        // The ideal way is to get the multisig account signer but it is unavailable right now. So the pattern is to
+        // create the metadata object using creator addresss as a partial seed and transfer it to multisig account.
+        object::transfer(creator, object::object_from_constructor_ref<ObjectCore>(constructor_ref), multisig_address);
+
+        // You should customize those arguments per your need.
+        primary_fungible_store::create_primary_store_enabled_fungible_asset(
+            constructor_ref,
+            option::none(), // maximum supply
+            utf8(b"meme coin"),
+            utf8(b"meme"),
+            8,
+            utf8(b"http://meme.xyz/favicon.ico"),
+            utf8(b"http://meme.xyz/"),
+        );
+
+        // Create mint/burn/transfer refs to allow creator to manage the fungible asset.
+        let mint_ref = fungible_asset::generate_mint_ref(constructor_ref);
+        let burn_ref = fungible_asset::generate_burn_ref(constructor_ref);
+        let transfer_ref = fungible_asset::generate_transfer_ref(constructor_ref);
+        let metadata_object_signer = object::generate_signer(constructor_ref);
+        move_to(
+            &metadata_object_signer,
+            ManagingRefs { mint_ref, transfer_ref, burn_ref }
+        )
+    }
+
+    #[view]
+    /// Return the address of the managed fungible asset that's created when this module is deployed.
+    public fun get_metadata(): Object<Metadata> {
+        let asset_address = object::create_object_address(&@example_addr, ASSET_SYMBOL);
+        object::address_to_object<Metadata>(asset_address)
+    }
+
+    /// Mint as the owner of metadata object to the primary fungible store of `to` account.
+    public entry fun mint_to_primary_store(admin: &signer, to: address, amount: u64) acquires ManagingRefs {
+        let asset = get_metadata();
+        let receiver_primary_store = primary_fungible_store::ensure_primary_store_exists(to, asset);
+        mint(admin, receiver_primary_store, amount);
+    }
+
+    public entry fun mint(
```

Comment:
This kinda seems like just a copy paste of managed_coin. May be we can just add a step in the tutorial that shows creating the multisig account separately and passing its address as the admin in the managed_fungible_asset module?

Reply:
There is a caveat. Your multisig account doesn't return a signer of that acct. So the `init_module` of managed_coin doesn't work in this case.